app/views/admin/splashpages/_form.html.haml

```diff
@@ -8,6 +8,10 @@
     .col-md-12
       = f.inputs name: 'Components' do
         %ul.fa-ul
+          %li
+            %th
+              %button.btn#selectAll{type: "button"}
```

Comment:
Just the `btn` class is not enough. Please read the [bootstrap documentation](https://getbootstrap.com/docs/3.4/css/#buttons).

Reply:
I would suggest `btn-default` as styling class. What do you think?

Reply:
I had tried that, it just looked fine to me without adding the styling class as I was a bit confused about how to style it, `btn-default` is also a good option. Thanks.

Reply:
@snpd25 apart from looking good, it has to look as the rest of the app :wink: